plugins/unix-socket.js

```diff
@@ -0,0 +1,97 @@
+var toPull = require('stream-to-pull-stream')
+var net = require('net')
+var fs = require('fs')
+var path = require('path')
+
+// hax on double transform
+var started = false
+
+module.exports = function (opts) {
+  const config = require('ssb-config/inject')(process.env.ssb_appname)
+  const socket = path.join(config.path, 'socket')
```

Comment:
would it be okay if just passed in `{path: absolute_path}` this module shouldn't be coupled to ssb.